aviary/modules/annotation/annotation.smk

```diff
@@ -163,7 +163,6 @@ rule checkm2:
     resources:
         mem_mb = lambda wildcards, attempt: min(int(config["max_memory"])*1024, 128*1024*attempt),
         runtime = lambda wildcards, attempt: 8*60*attempt,

```

Comment:
I'm assuming that this flag wasn't doing anything and checkm2 will just use GPUs if the system has them?